NavigationReact/sample/relay-modern/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "navigation-relay",
+  "description": "Navigation React Relay (modern) example",
```

Comment:
I think we should rename the package to "navigation-relay-modern" and change the description to "Navigation React Relay Modern example". What do you think?